source/PowerUp.js

```diff
@@ -377,7 +377,7 @@ export class BFG extends PowerUp {
       constructor() {
           super()
           this.vector = new Victor(1,0)
-          this.bulletSpeed = 2
+          this.bulletSpeed = 3
```

Comment:
In case we ever need to tweak this again, would it be worth it to pull these out to `Reference.NORMAL_BULLET_SPEED` and `Reference.FAST_BULLET_SPEED`?